pdc-client.spec

```diff
@@ -147,7 +147,7 @@ cat > %{buildroot}/%{_sysconfdir}/pdc.d/fedora.json << EOF
     "fedora": {
         "host": "https://pdc.fedoraproject.org/rest_api/v1/",
         "develop": false,
-        "insecure": false
+        "ssl-verify": true
```

Comment:
I'm not experienced writing RPM spec files; can any section be written in Python, to make JSON access easier? Maybe it would be best if I modified the source here to accept both values from the configuration, and emit a warning about a deprecated option?